test/fenv/run_fenv_test.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+sst --add-lib-path=../../build/src/ ../rev-model-options-config.py -- --program="$1" --verbose=0 | awk 'BEGIN {pass=0} /Simulation is complete/ {pass=1;next} {print} END {exit !pass}'
```

Comment:
The AWK script detects and filters out `Simulation is complete`, and returns a nonzero exit code if `Simulation is complete` was not seen, which becomes the exit code of the script. If it fails, the parallel `make` fails, and `Simulation is complete` is never printed by `make`.

Reply:
I tried to make the exit code of the script be the sole determinant of whether the test passed or failed, but it introduced too many problems, so I had to fall back on this expected regex in output. The AWK script detects it in tests and turns it into an exit code for a build; then the parallel make passes or fails if all of its parallel targets pass. If all pass, the parallel make prints the message which `ctest` expects, and the test passes.